docs/msbuild/property-functions.md

```diff
@@ -241,7 +241,7 @@ $[MSBuild]::GetDirectoryNameOfFileAbove(string ThePath, string TheFile)
  This property function has the following syntax:  
 
 ```  
-$([MSBuild]::GetPathOfFileAbove(dir.props)  
+$([MSBuild]::GetPathOfFileAbove(dir.props))  
```

Comment:
Or should the first parentheses be removed instead, like the GetDirectoryNameOfFileAbove example?

Reply:
No, but good catch: I have added https://github.com/MicrosoftDocs/visualstudio-docs/pull/652/commits/f04ecfab317dd89135c57a55f14c603a3900a69f to this PR to fix additional missing parentheses in the examples for `DoesTaskHostExist`, `GetDirectoryNameOfFileAbove` and `MakeRelative`.